src/storage/mongodb/CarStorage.ts

```diff
@@ -143,164 +144,16 @@ export default class CarStorage {
-          chargePhaseAmp: Utils.convertToInt(chargeOptionTables.chargePhaseAmp),
-          chargePhase: Utils.convertToInt(chargeOptionTables.chargePhase),
-          chargePower: Utils.convertToFloat(chargeOptionTables.chargePower),
-          chargeTime: Utils.convertToInt(chargeOptionTables.chargeTime),
-          chargeSpeed: Utils.convertToInt(chargeOptionTables.chargeSpeed)
-        })) : [],
-      fastChargePlug: carToSave.fastChargePlug,
-      fastChargePlugEstimate: carToSave.fastChargePlugEstimate,
-      fastChargePlugLocation: carToSave.fastChargePlugLocation,
-      fastChargePowerMax: Utils.convertToInt(carToSave.fastChargePowerMax),
-      fastChargePowerAvg: Utils.convertToInt(carToSave.fastChargePowerAvg),
-      fastChargeTime: Utils.convertToInt(carToSave.fastChargeTime),
-      fastChargeSpeed: Utils.convertToInt(carToSave.fastChargeSpeed),
-      fastChargeOptional: carToSave.fastChargeOptional,
-      fastChargeEstimate: carToSave.fastChargeEstimate,
-      batteryCapacityUseable: Utils.convertToFloat(carToSave.batteryCapacityUseable),
-      batteryCapacityFull: Utils.convertToFloat(carToSave.batteryCapacityFull),
-      batteryCapacityEstimate: carToSave.batteryCapacityEstimate,
-      dimsLength: Utils.convertToInt(carToSave.dimsLength),
-      dimsWidth: Utils.convertToInt(carToSave.dimsWidth),
-      dimsHeight: Utils.convertToInt(carToSave.dimsHeight),
-      dimsWheelbase: carToSave.dimsWheelbase,
-      dimsWeight: Utils.convertToInt(carToSave.dimsWeight),
-      dimsBootspace: Utils.convertToInt(carToSave.dimsBootspace),
-      dimsBootspaceMax: Utils.convertToInt(carToSave.dimsBootspaceMax),
-      dimsTowWeightUnbraked: Utils.convertToInt(carToSave.dimsTowWeightUnbraked),
-      dimsTowWeightBraked: Utils.convertToInt(carToSave.dimsTowWeightBraked),
-      dimsRoofLoadMax: Utils.convertToInt(carToSave.dimsRoofLoadMax),
-      miscBody: carToSave.miscBody,
-      miscSegment: carToSave.miscSegment,
-      miscSeats: Utils.convertToInt(carToSave.miscSeats),
-      miscRoofrails: carToSave.miscRoofrails,
-      miscIsofix: carToSave.miscIsofix,
-      miscIsofixSeats: carToSave.miscIsofixSeats,
-      miscTurningCircle: carToSave.miscTurningCircle,
-      euroNCAPRating: Utils.convertToInt(carToSave.euroNCAPRating),
-      euroNCAPYear: carToSave.euroNCAPYear,
-      euroNCAPAdult: Utils.convertToInt(carToSave.euroNCAPAdult),
-      euroNCAPChild: Utils.convertToInt(carToSave.euroNCAPChild),
-      euroNCAPVRU: Utils.convertToInt(carToSave.euroNCAPVRU),
-      euroNCAPSA: Utils.convertToInt(carToSave.euroNCAPSA),
-      relatedVehicleIDSuccessor: Utils.convertToInt(carToSave.relatedVehicleIDSuccessor),
-      eVDBDetailURL: carToSave.eVDBDetailURL,
-      videos: carToSave.videos,
-      hash: carToSave.hash,
-      imageURLs: carToSave.imageURLs,
-      image: carToSave.image,
-      imagesHash: carToSave.imagesHash
+      _id: carToSave.id,
+      ...carCatalog,
```

Comment:
',' after carCatalog should be removed

Reply:
Thanks for your feedback, we usually let the ending comma to ease the cut'n paste in a Json document :-)